cola/libdialect/constraints.cpp

```diff
@@ -112,6 +115,9 @@ SepDir dialect::cardinalDirToSepDir(CardinalDir dir) {
         return SepDir::WEST;
     case CardinalDir::NORTH:
         return SepDir::NORTH;
+    default:
+        COLA_ASSERT(false);
+        return SepDir::EAST;
```

Comment:
Same comment. Wherever it's possible to get away without returns (in this and other files), I'd prefer to.